app/services/api/v1/authenticate_user.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Api::V1::AuthenticateUser
+  prepend SimpleCommand
+
+  def initialize(user, account)
+    @user    = user
+    @account = account
+  end
+
+  def call
+    if @user && @account
+      JsonWebToken.encode({ user_id: @user.id, account_id: @account.id })
```

Comment:
```suggestion
      JsonWebToken.encode user_id: @user.id, account_id: @account.id
```
Esto es _a ojo_, pero casi seguro funciona.

Reply:
funciona perfecto franco, que loco esto porque yo pienso que es el segundo parametro del metodo, pero claro como le pasas la key sabe que es un hash jajajaj